### PR TITLE
Detect new brands: MeMobile, VOLIA and Detect device for exists brands

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: Parser/Device/AbstractDeviceParser.php [hunk 1]
```diff
@@ -992,6 +992,7 @@ abstract class AbstractDeviceParser extends AbstractParser
         'M3'  => 'Mecer',
         'M3M' => 'M3 Mobile',
         '0M'  => 'Mecool',
+        'MEM' => 'Me Mobile',
```
Comment (Simbiat) [inline Parser/Device/AbstractDeviceParser.php:995]: It looks like official name is `MeMobile" without a space. At least the closet to official seems to be their page https://www.facebook.com/memobilepakistan/ (their website is not loading for me). Media seems to be mostly using it with space, but I've found some pages without it, too.

Comment (Simbiat) [reply Parser/Device/AbstractDeviceParser.php:995]: also youtube channel https://www.youtube.com/channel/UC-FA0BPxjtVhkfg0IknOawg

Comment (Simbiat) [reply Parser/Device/AbstractDeviceParser.php:995]: Huh, their official website did load for after like 5 minutes of loading (per page), and I finally got to http://www.memobile.pk/legal/ where they are not using space in the name. I would consider legal name to be the "correct" one.

Comment (sanchezzzhak) [reply Parser/Device/AbstractDeviceParser.php]: fix

**Review by sgiehl — APPROVED** [review]

**Pull request merged**